src/notifications/types.ts

```diff
@@ -55,6 +56,7 @@ export type NotificationSourceRequestContext = NotificationBaseContext & {
 
 export type NotificationDoneByContext = NotificationBaseContext & {
   doneBy: Reference<User>;
+  doneTo?: Reference<User>;
```

Comment:
What's the purpose of this field? Isn't that always the notification user?

Reply:
Yes, but we only have the `userId` of the user to notify in this context but we need the username. Or do we have a pre-existing property for the relevant User object?